src/main/java/kevin/Kevin.java

```diff
@@ -0,0 +1,56 @@
+package kevin;
+
+import kevin.evaluator.Evaluator;
+import kevin.parser.Parser;
+import kevin.parser.FileParser;
+import kevin.parser.QueryObject;
+import kevin.ui.Logger;
+import kevin.exception.KevinException;
+import kevin.storage.FileStorage;
+import kevin.taskList.TaskList;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Kevin class, a task list chatbot.
+ */
+public class Kevin {
+    public static void main(String[] args) {
+        Logger logger = new Logger();
+        Parser parser = new Parser();
+        FileParser fileParser = new FileParser();
+        FileStorage fileStorage = new FileStorage();
+        TaskList taskList = new TaskList();
+        Scanner scanner = new Scanner(System.in);
+        Evaluator evaluator = new Evaluator(logger, fileStorage, taskList);
```

Comment:
I like how you abstracted different types of objects and organised them. Any reason why you did it this way instead of minimising the number of classes?